src/config.ts

```diff
@@ -16,14 +16,11 @@ export default {
     SALTYRTC_SERVER_KEY: 'b1337fc8402f7db8ea639e05ed05d65463e24809792f91eca29e88101b4a2171',
 
     // ICE
-    SALTYRTC_STUN: {
-        urls: 'stun:stun.threema.ch:443',
-    },
-    SALTYRTC_TURN: {
-        urls: 'turn:turn.threema.ch:443',
+    ICE_SERVERS: [{
+        urls: ['turn:turn.threema.ch:443', 'turn:turn.threema.ch:443?transport=tcp'],
```

Comment:
Do you think the first URL should be explicitly marked with `?transport=udp`?

[RFC7065](https://tools.ietf.org/html/rfc7065) does not specify what protocol to use when not specifying a transport explicitly. So it must be up to the browser.